runtime/compiler/net/ServerStream.cpp

```diff
@@ -72,7 +72,7 @@ SSL_CTX *createSSLContext(TR::PersistentInfo *info)
       exit(1);
       }
 
-   SSL_CTX_set_session_id_context(ctx, (const unsigned char*) "JITaaS", 6);
+   SSL_CTX_set_session_id_context(ctx, (const unsigned char*) "JITServer", 9);
```

Comment:
I would have missed that 9. Maybe it's better to use strlen(JITServer) and hope that the c++ compiler will know to optimize is away.